src/main/java/com/powsybl/network/conversion/server/NetworkConversionObserver.java

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.network.conversion.server;
+
+import com.powsybl.iidm.network.Network;
+import com.powsybl.network.conversion.server.dto.ExportNetworkInfos;
+import io.micrometer.core.instrument.Counter;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.observation.Observation;
+import io.micrometer.observation.ObservationRegistry;
+import lombok.NonNull;
+import org.springframework.stereotype.Service;
+
+/**
+ * @author Anis Touri <anis.touri at rte-france.com>
+ */
+
+@Service
+public class NetworkConversionObserver {
+
+    private static final String OBSERVATION_PREFIX = "app.conversion.";
+    private static final String CONVERSION_FAILED_NAME = OBSERVATION_PREFIX + "failed";
```

Comment:
Rename by FAILED_CONVERSION_COUNTER_NAME
Need to distinguish import and export ?